tests/functional/complete_sign_in.js

```diff
@@ -60,7 +41,18 @@ define([
       user = TestHelpers.emailToUser(email);
       return this.remote
         .then(clearBrowserState())
-        .then(setupTest(this, true));
+        .then(createUser(email, PASSWORD, { preVerified: true }))
+        .then(openPage(PAGE_SIGNIN_URL, '#fxa-signin-header'))
+        .execute(listenForFxaCommands)
+        .then(fillOutSignIn(email, PASSWORD))
+        .then(testIsBrowserNotified('can_link_account'))
+        .then(testIsBrowserNotifiedOfLogin(email, { expectVerified: false }))
+        .then(testElementExists('#fxa-confirm-signin-header'))
+        .then(restmail(EMAIL_SERVER_ROOT + '/mail/' + user))
+        .then(function (emails) {
```

Comment:
Do we prefer fat arrows where possible, or are we not fussed about it? (I'll stop mentioning it if it's the latter)